tests/test_scenario.py

```diff
@@ -548,6 +548,149 @@ def test_clone(self, session, network_with_data):
 
         return updated_network
 
+    def test_get_inherited_data(self, session, network_with_child_scenario):
```

Comment:
Can we add a test with a grand-parent to test recursion past the first level?

Reply:
Done